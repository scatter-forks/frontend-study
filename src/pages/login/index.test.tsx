import { shallow } from 'enzyme'
import LoginPage from './index'
const { configure } = require('enzyme')
const Adapter = require('enzyme-adapter-react-16')
configure({ adapter: new Adapter() })

describe('test LoginPage', () => {
    let props: any
    beforeEach(() => {
        props = {
            history: {
                push: jest.fn()
            }
        }
    })
    
    it('snapshot', () => {
        const wrapper = shallow(<LoginPage {...props}/>)
        expect(wrapper).toMatchSnapshot()
    })
})