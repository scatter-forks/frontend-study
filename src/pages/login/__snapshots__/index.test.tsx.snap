// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test LoginPage snapshot 1`] = `ShallowWrapper {}`;
